modules/page_object_about_prefs.py

```diff
@@ -1,5 +1,7 @@
 from pypom import Region
+from selenium.webdriver import Firefox
 from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
```

Comment:
Q: why the new imports? They don't _appear_ to be used, unless I'm missing some trick (disclaimer: I'm pretty Selenium/Python ignorant).

Reply:
My bad, will fix